token-lending/sdk/src/state/obligation.rs

```diff
@@ -37,9 +37,14 @@ pub struct Obligation {
     pub borrows: Vec<ObligationLiquidity>,
     /// Market value of deposits
     pub deposited_value: Decimal,
-    /// Risk-adjusted market value of borrows
+    /// Risk-adjusted market value of borrows.
+    /// ie sum(b.borrowed_amount * b.current_spot_price * b.borrow_weight for b in borrows)
     pub borrowed_value: Decimal,
-    /// The maximum borrow value at the weighted average loan to value ratio
+    /// Risk-adjusted upper bound market value of borrows.
+    /// ie sum(b.borrowed_amount * max(b.current_spot_price, b.smoothed_price) * b.borrow_weight for b in borrows)
+    pub borrowed_value_upper_bound: Decimal,
+    /// The maximum borrow value at the weighted average loan to value ratio using the minimum of
```

Comment:
the maximum borrow value using min of spot and smooth price times ltv of each asset

something closer to this?